docs/DataConsistencyCheck.md

```diff
@@ -0,0 +1,99 @@
+# Table of Contents
+- [Description](../README.md#description)
+- [Getting Started](../README.md#getting-started)
+    - [Build Application](../README.md#build-application)
+    - [Basic Run Configuration](../README.md#basic-run-configuration)
+- [Changes](../README.md#changes)
+- [Usage](Usage.md)
+    - [Declaring Test Runs](Usage.md#declaring-test-runs)
+    - [Configuration](Usage.md#configuration)
+    - [Parameters](Usage.md#parameters)
+    - [Originating Identity](Usage.md#originating-identity)
+    - [Declaring Services](Usage.md#declaring-services)
+- Test Classes
+    - [Catalog](CatalogTest.md)
+       - [Example Output](CatalogTest.md#example-output)
+    - [Provision](ProvisionTests.md#provision-tests)
+        - [Test Procedure](ProvisionTests.md#test-procedure)
+        - [Version specific Tests](ProvisionTests.md#version-specific-tests)
+        - [Example Output](ProvisionTests.md#example-output)
+    - [Binding](#binding-tests)
+        - [Test Procedure](BindingTests.md#test-procedure)
+        - [Version specific Tests](BindingTests.md#version-specific-tests)
+        - [Example Output](BindingTests.md#example-output)
+    - [Authentication](docs/AuthenticationTests.md)   
+    - [Contract](docs/ContractTest.md)
+    - [Data Consistency Check](#data-consistency-check)
+        - [Test Procedure](#test-procedure)
+        - [Example Output](#example-output)
+- [Contribution](docs/Contribution.md)
+   
+# Data Consistency Check
+
+The Data Consistency Check tests if creating instances and bindings with the same instanceId or bindingId,
+works, if the service instance got deleted. The goal is to ensure that all binding on a service don't remain 
+in the database of the service broker. This is done for all plans in the catalog. bindings and provisions are created based upon the 
+api-version and configuration like in [Binding Tests](BindingTests.md).
+
+## Test Procedure
+
+- Valid Provision and Bindings
+    - Create a valid provision.
+        - If the Service broker creates service instances asynchronously, the checker will start polling and and verify the responses.
+        - When configured to do so, the checker verifies if the dashboard URL works.
+        are according to [spec](https://github.com/openservicebrokerapi/servicebroker/blob/v2.15/spec.md#polling-last-operation-for-service-instances) and finish successfully.
+        - Test what happens when attempting to create a service instances with the same instance id and same parameters and with different parameters.
```

Comment:
'a' and plural are not working together. I think you mean service instance instead of instances